contrib/openstack/README.md

```diff
@@ -1,3 +1,16 @@
-# Provision a Deis Cluster on OpenStack
+# Provision a Deis Cluster on Openstack
 
 Please refer to the instructions at http://docs.deis.io/en/latest/installing_deis/openstack/.
+
+Meanwhile here is how I've been testing it:
+
+```
+$ make discovery-url
+$ terraform apply \
+      -var "username=$OS_USERNAME" \
+      -var "password=$OS_PASSWORD" \
+      -var "tenant=$OS_TENANT_NAME" \
+      -var "auth_url=$OS_AUTH_URL" \
+      -var "image=coreos-717.3.0" \
```

Comment:
Could you update this to 766.4.0 (or whatever is current in https://github.com/deis/deis/blob/master/contrib/utils.sh#L16)?